### ISSUE TITLE
Request: add fields to layout tab without dragging

### ISSUE THREAD

**Issue by michaeldhopkins** [craftcms/cms#3047]
Hi, I have a feature request. We use a lot of field groups, enough that the tab is offscreen and the drag distance from some of the fields to the layout tab is considerable or even outside the viewport (see screenshot.) 

I'd like double-clicking a field to instantly add the field to the layout tab as an alternative to dragging. Double-click to move something out of a draggable list is a common interactive method and I don't believe it would surprise users. It was used in EE Playa, IIRC.  If two or more layout tabs are available, defaulting to the leftmost would work. A short drag-and-drop to reposition after a double-click would still save time.  

Thanks for considering this feature request!

<img width="1244" alt="screen shot 2018-06-28 at 3 05 58 pm" src="https://user-images.githubusercontent.com/133994/42060845-d10c2a32-7aed-11e8-87c2-1077eb78dd40.png">

**Comment by davidrapson** [comment]
+1 to this, particularly the bug described in https://github.com/craftcms/cms/issues/3088

With more than a few field groups the only way to add most fields to a tab is to zoom waay out and then drag and drop that way. Anyway of making this more manageable would be great.

**Comment by brandonkelly** [comment]
We’ve got some field layout improvements slated for 3.2 ;)

**Comment by KatieMFritz** [comment]
Seems like this would be an accessibility improvement too. I haven't tried to configure a field layout with only a keyboard, but I'm guessing it's not possible?

**Comment by brandonkelly** [comment]
@KatieMFritz Yeah not currently, but the new layout designer should be fully keyboard accessible.
